distro/configuration/patientidentifiertypes/pit.csv

```diff
@@ -3,5 +3,5 @@ Uuid,Void/Retire,Name,Description,Required,Format,Format description,Validator,L
 22348099-3873-459e-a32e-d93b17eda533,,Legacy ID,Identifier type to record optional previous identifers ,FALSE,,,,NOT_USED,
 b4143563-16cd-4439-b288-f83d61670fc8,,ID Card,ID Card,FALSE,,,,NOT_USED,UNIQUE
 a71403f3-8584-4289-ab41-2b4e5570bd45,,SSN,Social Security Number,FALSE,[A-Z]{1}-[0-9]{7},"Identifier should be 1 letter, followed by a dash and 7 numerical characters. Eg, A-0010902",,NOT_USED,UNIQUE
-e6baf185-38ed-4815-9476-f98d2cc2b331,Unique ART NUmber,Unique ART Number,TRUE,TRUE,"^(?:TI-)?[A-Z]{2,3}\/[A-Z]{3}\/0*\d{5,5}$",The patient identifier should match the following format "CES/JHT/00005" or "TI-CES/JHT/00005" for transfer in clients,NOT_USED,UNIQUE,UNIQUE
+e6baf185-38ed-4815-9476-f98d2cc2b331,Unique ART NUmber,Unique ART Number,TRUE,TRUE,[0-9]{5},The patient identifier should match the following format "00005" for transfer in clients,NOT_USED,UNIQUE,UNIQUE
```

Comment:
Why is this update necessary? @Michaelndula

Reply:
@moshonk this is resoved